akka-javasdk/src/main/java/akka/javasdk/agent/ConversationMemory.java

```diff
@@ -85,15 +93,15 @@ public State emptyState() {
   public sealed interface Event {
 
     @TypeName("akka-memory-limited-window-set")
-    record LimitedWindowSet(int maxSize) implements Event {
+    record LimitedWindowSet(int maxSizeInBytes) implements Event {
     }
 
     @TypeName("akka-memory-user-message-added")
-    record UserMessageAdded(String message) implements Event {
+    record UserMessageAdded(String componentId, String message) implements Event {
```

Comment:
and the tokens would be added to the event too, and preferably the event should also include a historyTokenCount (total), which can be a trigger for compaction from a consumer, or display in a view.

btw, historySize would also be useful in the event, to trigger compaction from a consumer

Reply:
added now as well. events include a `tokenTotalUsage`